dahu/core/app/scripts/views/workspace/objects/tooltip.js

```diff
@@ -34,6 +35,10 @@ define([
 
         className: 'object tooltip ui-widget-content',
 
+        initialize: function() {
+            this.changeBackgroundColor();
```

Comment:
the default color is in the tooltip model if I refer to dahu/core/app/scripts/models/objects/tooltip.js it should not be in the CSS